### DISCUSSION TITLE
possible DNS issue with talos initial cluster configuration

### DISCUSSION THREAD

**Discussion by matjazantloga** [siderolabs/talos#9183] [Q&A] [answered]
i'm installing 3 control planes for start in virtual enviroment.

i've came so far

[ma@localhost vmamd1]$ talosctl config endpoint 10.99.100.132 nodes 10.99.100.132 10.99.100.133 10.99.100.134
talosctl config contexts
CURRENT   NAME            ENDPOINTS                                                       NODES
*         TalosThinkPAD   10.99.100.132,nodes,10.99.100.132,10.99.100.133,10.99.100.134

but when i run sth like that

[ma@localhost vmamd1]$ talosctl get members -n 10.99.100.132
rpc error: code = Unavailable desc = last connection error: connection error: desc = "transport: Error while dialing: dial tcp: lookup nodes on 1.1.1.1:53: no such host"

i got dns issue. 

on machine with talosctl and on all talos nodes i'm using ext dns 1.1.1.1
should i switch on internal DNS and prepare a zone file ?

**Comment by smira** [comment, marked as answer]
`talosctl config endpoint 10.99.100.132 nodes 10.99.100.132 10.99.100.133 10.99.100.134` this command is wrong

It should be:

`talosctl config endpoint 10.99.100.132`

There's nothing to be done after that, just use `talosctl -n <node_IP> get members`

**Reply by matjazantloga** [reply]
i'm still issue
![image](https://github.com/user-attachments/assets/7326b757-3506-4907-8365-c1f769837b44)


controlplane.yaml , part for pod & service subnets

![image](https://github.com/user-attachments/assets/4eddfec1-720c-4f56-8370-1d6e0f3ccc11)

**Comment by matjazantloga** [comment]
finally got cluster running  :)

![image](https://github.com/user-attachments/assets/567a8156-1223-4aef-95dd-4a5871c1c9aa)
